codespell_lib/data/dictionary.txt

```diff
@@ -15899,6 +15911,7 @@ iresistibly->irresistibly
 iritable->irritable
 iritated->irritated
 ironicly->ironically
+irradation->irradiation
```

Comment:
also irradate->irradiate ?

(and perhaps also iradiate, iradate, iradiation, iradation etc.)

Could maybe also add `irigation->irrigation` (etc.) if you're feeling enthusiastic? :wink: 

And I've just spotted (sorry!) that the dictionary has `iritated->irritated` but not `iritate->irritate`.

Reply:
added stuff, except irigat*